iothub_client/src/iothub_client_core.c

```diff
@@ -43,6 +46,8 @@ typedef struct IOTHUB_CLIENT_CORE_INSTANCE_TAG
     struct IOTHUB_QUEUE_CONTEXT_TAG* connection_status_user_context;
     struct IOTHUB_QUEUE_CONTEXT_TAG* message_user_context;
     struct IOTHUB_QUEUE_CONTEXT_TAG* method_user_context;
+    uint16_t do_work_freq_ms;
+    tickcounter_ms_t currentMessageTimeout;
```

Comment:
>tickcounter_ms_t [](start = 4, length = 16)

This should probably be just a uint32_t instead of a tickcounter_ms_t.  

As a side note if you bring in a value you need to make sure you add the #include header at the top.